adapter/api/src/main/java/org/apache/rocketmq/eventbridge/adapter/api/handler/WebLogAspect.java

```diff
@@ -18,35 +18,45 @@
 package org.apache.rocketmq.eventbridge.adapter.api.handler;
 
 import com.google.gson.Gson;
-import lombok.extern.slf4j.Slf4j;
 import org.aspectj.lang.ProceedingJoinPoint;
 import org.aspectj.lang.annotation.Around;
 import org.aspectj.lang.annotation.Aspect;
 import org.aspectj.lang.annotation.Pointcut;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
+import reactor.core.publisher.Mono;
+
+import java.util.Objects;
 
-@Slf4j
 @Aspect
 @Component
 public class WebLogAspect {
 
+    private static final Logger log = LoggerFactory.getLogger("accessLog");
+
     @Pointcut("@annotation(org.apache.rocketmq.eventbridge.adapter.api.annotations.WebLog)")
     public void webLog() {
 
     }
 
     @Around("webLog()")
```

Comment:
Annotations are not recommended for access logs, and unmarked requests should also have access logs.

Reply:
filter add access log